lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -130,19 +130,12 @@ abstract class SqlParserTestBase : TestBase() {
      * a [PartiqlAst] and to an IonValue Sexp equals the [expectedSexpAst]. Next checks that converting this IonValue
      * Sexp to an ExprNode equals the [actualExprNode].
      */
-    private fun partiqlAssert(actualExprNode: ExprNode, expectedIonSexp: IonSexp, source: String) {
-        val actualStatment = parseToAst(source)
+    private fun checkEqualInIonSexp(actualExprNode: ExprNode, expectedIonSexp: IonSexp, source: String) {
+        val actualStatment = actualExprNode.toAstStatement()
```

Comment:
`partiqlAssert` asserts on `IonSexp` as well as `ExprNode`. Shouldn't the new method also assert on `ExprNode`? We should be careful while removing such `asserts` as it might cause some breaking changes go unnoticed.

Reply:
Nvm. I just saw that `pigExprNodeTransformAsserts` takes care of this.